functions/src/utils/venue.js

```diff
@@ -3,18 +3,11 @@ const { VenueIdSchema } = require("../types/venue");
 const VENUE_ID_REGEX = /[a-z0-9_]{1,250}/;
 const INVALID_SLUG_CHARS_REGEX = /[^a-zA-Z0-9]/g;
 
-/**
- * @deprecated Use generateSlug instead.
- */
-const getVenueId = (name) =>
-  name.replace(INVALID_SLUG_CHARS_REGEX, "").toLowerCase();
```

Comment:
This function was misleading as slug is not always going to be the ID of a venue.